app/views/templates/formatters/links/btaNavigationLinks.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
```

Comment:
Could you move this file into the `/views/templates` directory? I'm thinking it's not technically a "formatter".